overlord/auth/auth.go

```diff
@@ -82,6 +84,17 @@ func (u *UserState) HasStoreAuth() bool {
 	return u.StoreMacaroon != ""
 }
 
+// HasExpired returns true if the user has an expiration set and
+// the expiration date is past the current time.
```

Comment:
```suggestion
// the expiration date is earlier than the current time.
```
("past" as a preposition usually means "after", so this is ambiguous)

Reply:
I think it's probably clearer switched around:

"... and we are paste the expiration date"
"... and current time is past the expiration date"